include/NeoFOAM/fields/domainField.hpp

```diff
@@ -78,6 +78,8 @@ class DomainField
 
     BoundaryFields<ValueType>& boundaryField() { return boundaryFields_; }
 
+    [[nodiscard]] const Executor& exec() const { return exec_; }
```

Comment:
Does not need to be nodiscard. We decided not to use [[nodiscard]] for trivial getter.
```suggestion
     const Executor& exec() const { return exec_; }
```